app/src/routes/(auth)/manager/sendConnectionEmail/+server.ts

```diff
@@ -53,23 +54,26 @@ export const POST: RequestHandler = async ({ request }) => {
 		.toPromise();
 
 	if (err) {
-		console.error('sendConnectionEmail', err);
+		logger.error(err, 'sendConnectionEmail');
 		throw error(500, 'sendConnectionEmail: server error');
 	}
 
 	if (!data.account) {
+		logger.error('sendConnectionEmail: account not found');
 		throw error(500, 'sendConnectionEmail: account not found');
 	}
 
 	if (!accountHasRelation(data.account)) {
+		logger.error('sendConnectionEmail: Invalid account type');
 		throw error(500, 'sendConnectionEmail: Invalid account type');
```

Comment:
nitpick: (désolé, c'est mon côté psycho-rigide) les 2 autres messages précédents commencent par une minuscule, celui-ci par une majuscule.

```suggestion
		throw error(500, 'sendConnectionEmail: invalid account type');
```